File: chaeryeon823/PGS_42747.js
// H-index
// Lv 2
// 정렬

function solution(citations) {
  let answer = 0;
  citations = citations.sort((a, b) => b - a);
  for (let i = 0; i < citations.length; i++) {
    if (i + 1 <= citations[i]) {
      answer = i + 1;
    } else {
      return answer;
    }
  }
  return answer;
}

File: chaeryeon823/PGS_42747.py
# H-Index
# lv 2
# 정렬
def solution(citations):
    answer = 0
    arr = sorted(citations, reverse = True)
    print(arr)
    for idx in range(len(arr)):
        if (idx + 1) <= arr[idx]:
            answer = idx + 1
        else:
            return answer
    
    return answer